bin/lando.js

```diff
@@ -7,95 +7,98 @@
  * @name lando
  */
 
-'use strict';
+ 'use strict';
 
 // Modules
-var _ = require('lodash');
-var bootstrap = require('./../lib/bootstrap.js');
-var cli = require('./../lib/cli');
-var log;
-var metrics;
-var os = require('os');
-var path = require('path');
-var Promise = require('./../lib/promise');
-var sudoBlock = require('sudo-block');
-var userConfRoot = path.join(os.homedir(), '.lando');
-var version = require(path.join(__dirname, '..', 'package.json')).version;
+const _ = require('lodash');
+const bootstrap = require('./../lib/bootstrap.js');
+const cli = require('./../lib/cli');
+const os = require('os');
+const path = require('path');
+const Promise = require('./../lib/promise');
+const sudoBlock = require('sudo-block');
+const userConfRoot = path.join(os.homedir(), '.lando');
+const version = require(path.join(__dirname, '..', 'package.json')).version;
+
+let log;
+let metrics;
 
 // Allow envvars to override a few core things
-var ENVPREFIX = process.env.LANDO_CORE_ENVPREFIX || 'LANDO';
-var LOGLEVELCONSOLE = process.env.LANDO_CORE_LOGLEVELCONSOLE || 'warn';
-var USERCONFROOT = process.env.LANDO_CORE_USERCONFROOT || userConfRoot;
+const ENVPREFIX = process.env.LANDO_CORE_ENVPREFIX || 'LANDO';
+const LOGLEVELCONSOLE = process.env.LANDO_CORE_LOGLEVELCONSOLE || 'warn';
+const USERCONFROOT = process.env.LANDO_CORE_USERCONFROOT || userConfRoot;
 
 // If we have CLI verbosity args let's use those instead
 if (cli.largv.verbose) { LOGLEVELCONSOLE = cli.largv.verbose + 1; }
 
 // Define the start up options
-var options = {
+const options = {
   configSources: [path.join(USERCONFROOT, 'config.yml')],
   envPrefix: ENVPREFIX,
   logLevelConsole: LOGLEVELCONSOLE,
   logDir: path.join(USERCONFROOT, 'logs'),
   mode: 'cli',
   pluginDirs: [USERCONFROOT],
   userConfRoot: USERCONFROOT,
-  version: version
+  version
 };
 
 // Error handler
-var handleError = function(error, log, metrics) {
+const handleError = ({hide, message, stack, code}, log, metrics) => {
```

Comment:
This looks like some object destructuring. I don't understand it much but I've avoided doing it thus far because of lack of understanding

Reply:
This isn't a blocker though, just a thought point

Reply:
I've read about it and I think I get it now.

This just takes all the parts of `error` that we used in the code, and explictly pulls those properties into the function rather than making you say `error.message` or `error.stack`.